### DISCUSSION TITLE
How to add navbar item with static site

### DISCUSSION THREAD

**Discussion by moonrailgun** [facebook/docusaurus#8402] [Q&A] [answered]
I have a static site build into `/static` folder(like try it online).

And i have no way to open it in navbar(because its alway think its a internel link and jump without native behavior). 

is any idea?

**Comment by slorber** [comment, marked as answer]
you can probably use our `pathname://` protocol to escape from the SPA navigation and navigate to static HTML files not known by Docusaurus:

https://docusaurus.io/docs/next/advanced/routing#escaping-from-spa-redirects
